src/effects/native/balanceeffect.cpp

```diff
@@ -1,7 +1,11 @@
 #include "balanceeffect.h"
 
-namespace {
+#include "util/defs.h"
 
+namespace {
+    double kMaxCorner = 22000; // Hz
```

Comment:
I have considered the same. Finally I found it funny to turn the gain for the right channel of and bring it back by the bypass freq. I am not sure if this is a valid use case. 
If you think not, I can lower the value. To which value?